core/admin/EE_Admin_Hooks.core.php

```diff
@@ -357,12 +358,26 @@ public function enqueue_scripts_styles()
     private function _set_defaults()
     {
         $this->_ajax_func = $this->_init_func = $this->_metaboxes = $this->_scripts = $this->_styles = $this->_wp_action_filters_priority = array();
-        $this->_current_route = isset($_REQUEST['action']) ? $_REQUEST['action'] : 'default';
+        $this->_current_route = $this->getCurrentRoute();
         $this->caller = get_class($this);
         $this->_extend = stripos($this->caller, 'Extend') ? true : false;
     }
 
 
+    /**
+     * A helper for determining the current route.
```

Comment:
I'd suggest giving a summary of the logic in the method description, so we can know what query parameters to set in order to set the current route without needing to read through the code. Eg

```suggestion
     * A helper for determining the current route. Uses `$_REQUEST['action']` if present and not set to `-1`, otherwise falls back to using `$_REQUEST['route']`.
```

Reply:
I'm not really sure how much the expanded comment in the header helps over reading the code and the comments in the code above the conditions.  I understand it's intended as a summary of the logic, however the summary isn't entirely accurate (because it doesn't include that it also considers whether `$_REQUEST['route']` is set).  Improving the summary essentially results in english form of the condition which I think would be superfluous imo, i.e.

> If an `action` argument is in the request and does not equal -1, then that value is considered the route.  Otherwise, the value for route will be the value of the `route` argument in the request if it exists.  Otherwise, the fallback value will be 'default'.

I don't think that gives much more helpful information than what is already concisely described in the actual conditions :)

Reply:
> I'm not really sure how much the expanded comment in the header helps over reading the code and the comments in the code above the conditions.

ya no worries, the idea was just to not require people to read the actual code and inline comments if they want to use that method. (IMO you want to learn how to use a method, you shouldn't need to read through its code and inline comments. You only do that if you're to learn *how* it works.)